auth/schema/mutations/register.py

```diff
@@ -33,15 +35,35 @@ def validate_email(email):
         if email_valid == False:
             raise ValidationError()
         return email
-        
+
+    '''Validate a username and make sure it contains only alphanumeric characters and special characters'''
+    def validate_username(username):
+        username_validator = RegexValidator(
+            r"(^[~0-9A-Za-z]*)",
+            "Username must contain only alphanumeric characters and special characters",
+            None,
+            re.IGNORECASE)
+        valid_username = username_validator.__call__(username)
```

Comment:
You do not have to call the `__call__` method. This is reserved magic method in python which makes the instances of the Class executable like a normal function. You can read up on python magic methods [here](https://www.python-course.eu/python3_magic_methods.php)
```python
 email_validator = EmailValidator()
        # email_valid = email_validator.__call__(email)
        email_valid = email_validator(email) # this would work better
```

Reply:
So where you used the `__call__` method, ensure you do not use it and execute the instance directly.